[2023-06-01T18:28:44.227+0000] {processor.py:157} INFO - Started process (PID=109) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:28:44.228+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:28:44.228+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:28:44.228+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:28:44.230+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:28:44.229+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:28:44.230+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:28:44.243+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.017 seconds
[2023-06-01T18:29:14.350+0000] {processor.py:157} INFO - Started process (PID=118) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:29:14.351+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:29:14.351+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:29:14.351+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:29:14.353+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:29:14.352+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:29:14.353+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:29:14.367+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.018 seconds
[2023-06-01T18:29:44.473+0000] {processor.py:157} INFO - Started process (PID=127) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:29:44.473+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:29:44.474+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:29:44.474+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:29:44.475+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:29:44.475+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:29:44.475+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:29:44.492+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.021 seconds
[2023-06-01T18:30:14.600+0000] {processor.py:157} INFO - Started process (PID=136) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:30:14.601+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:30:14.601+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:30:14.601+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:30:14.602+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:30:14.602+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:30:14.603+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:30:14.615+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.017 seconds
[2023-06-01T18:30:44.717+0000] {processor.py:157} INFO - Started process (PID=145) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:30:44.719+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:30:44.720+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:30:44.720+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:30:44.721+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:30:44.721+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:30:44.721+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:30:44.761+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.046 seconds
[2023-06-01T18:31:14.874+0000] {processor.py:157} INFO - Started process (PID=154) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:31:14.875+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:31:14.876+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:31:14.876+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:31:14.877+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:31:14.877+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:31:14.877+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:31:14.918+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:31:45.027+0000] {processor.py:157} INFO - Started process (PID=163) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:31:45.029+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:31:45.029+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:31:45.029+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:31:45.031+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:31:45.030+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:31:45.031+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:31:45.070+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:32:15.177+0000] {processor.py:157} INFO - Started process (PID=172) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:32:15.179+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:32:15.179+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:32:15.179+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:32:15.180+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:32:15.180+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:32:15.180+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:32:15.220+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:32:45.323+0000] {processor.py:157} INFO - Started process (PID=181) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:32:45.327+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:32:45.327+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:32:45.327+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:32:45.329+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:32:45.328+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:32:45.329+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:32:45.369+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.047 seconds
[2023-06-01T18:33:15.480+0000] {processor.py:157} INFO - Started process (PID=190) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:33:15.482+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:33:15.482+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:33:15.482+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:33:15.484+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:33:15.483+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:33:15.484+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:33:15.523+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:33:45.639+0000] {processor.py:157} INFO - Started process (PID=199) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:33:45.641+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:33:45.641+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:33:45.641+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:33:45.643+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:33:45.642+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:33:45.643+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:33:45.683+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:34:15.799+0000] {processor.py:157} INFO - Started process (PID=208) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:34:15.800+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:34:15.800+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:34:15.800+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:34:15.801+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:34:15.801+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:34:15.801+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:34:15.841+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.044 seconds
[2023-06-01T18:34:45.953+0000] {processor.py:157} INFO - Started process (PID=217) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:34:45.956+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:34:45.956+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:34:45.956+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:34:45.958+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:34:45.957+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:34:45.958+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:34:45.998+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.046 seconds
[2023-06-01T18:35:16.110+0000] {processor.py:157} INFO - Started process (PID=226) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:35:16.112+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:35:16.112+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:35:16.112+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:35:16.114+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:35:16.113+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:35:16.114+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:35:16.154+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:35:46.270+0000] {processor.py:157} INFO - Started process (PID=235) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:35:46.271+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:35:46.272+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:35:46.272+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:35:46.273+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:35:46.272+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:35:46.273+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:35:46.313+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:36:16.426+0000] {processor.py:157} INFO - Started process (PID=244) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:36:16.427+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:36:16.428+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:36:16.428+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:36:16.429+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:36:16.429+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:36:16.429+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:36:16.469+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:36:46.585+0000] {processor.py:157} INFO - Started process (PID=253) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:36:46.586+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:36:46.587+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:36:46.587+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:36:46.588+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:36:46.588+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:36:46.588+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:36:46.628+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:37:16.740+0000] {processor.py:157} INFO - Started process (PID=262) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:37:16.740+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:37:16.741+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:37:16.741+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:37:16.742+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:37:16.742+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:37:16.742+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:37:16.782+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.044 seconds
[2023-06-01T18:37:46.893+0000] {processor.py:157} INFO - Started process (PID=271) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:37:46.894+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:37:46.894+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:37:46.894+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:37:46.896+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:37:46.895+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:37:46.896+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:37:46.935+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:38:17.052+0000] {processor.py:157} INFO - Started process (PID=280) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:38:17.054+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:38:17.054+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:38:17.054+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:38:17.056+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:38:17.055+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:38:17.056+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:38:17.095+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:38:47.219+0000] {processor.py:157} INFO - Started process (PID=289) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:38:47.222+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:38:47.222+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:38:47.222+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:38:47.224+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:38:47.223+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 9, in <module>
    from .utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:38:47.224+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:38:47.264+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.047 seconds
[2023-06-01T18:39:00.324+0000] {processor.py:157} INFO - Started process (PID=291) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:39:00.326+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:39:00.326+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:39:00.326+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:39:00.328+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:39:00.328+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:39:00.328+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:39:00.347+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.024 seconds
[2023-06-01T18:39:42.826+0000] {processor.py:157} INFO - Started process (PID=23) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:39:42.827+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:39:42.827+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:39:42.827+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:39:42.829+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:39:42.828+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:39:42.829+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:39:42.854+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.030 seconds
[2023-06-01T18:40:12.952+0000] {processor.py:157} INFO - Started process (PID=32) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:40:12.954+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:40:12.954+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:40:12.954+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:40:12.958+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:40:12.957+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:40:12.958+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:40:12.975+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.026 seconds
[2023-06-01T18:40:43.080+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:40:43.081+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:40:43.082+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:40:43.082+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:40:43.083+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:40:43.083+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:40:43.083+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:40:43.123+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.044 seconds
[2023-06-01T18:41:13.228+0000] {processor.py:157} INFO - Started process (PID=50) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:41:13.229+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:41:13.229+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:41:13.229+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:41:13.231+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:41:13.231+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:41:13.231+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:41:13.271+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:41:43.385+0000] {processor.py:157} INFO - Started process (PID=59) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:41:43.386+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:41:43.386+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:41:43.386+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:41:43.388+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:41:43.388+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:41:43.388+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:41:43.428+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:42:11.527+0000] {processor.py:157} INFO - Started process (PID=68) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:42:11.528+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:42:11.528+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:42:11.528+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:42:11.531+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:42:11.530+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:42:11.531+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:42:11.551+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.026 seconds
[2023-06-01T18:42:47.247+0000] {processor.py:157} INFO - Started process (PID=23) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:42:47.248+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:42:47.248+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:42:47.248+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:42:47.250+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:42:47.249+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:42:47.250+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:42:47.270+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.025 seconds
[2023-06-01T18:43:17.377+0000] {processor.py:157} INFO - Started process (PID=32) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:43:17.378+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:43:17.379+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:43:17.379+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:43:17.380+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:43:17.380+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:43:17.380+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:43:17.420+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.046 seconds
[2023-06-01T18:43:47.531+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:43:47.553+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:43:47.553+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:43:47.553+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:43:47.555+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:43:47.554+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:43:47.555+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:43:47.576+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.046 seconds
[2023-06-01T18:44:17.683+0000] {processor.py:157} INFO - Started process (PID=50) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:44:17.684+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:44:17.684+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:44:17.684+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:44:17.686+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:44:17.686+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:44:17.686+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:44:17.726+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:44:47.832+0000] {processor.py:157} INFO - Started process (PID=59) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:44:47.834+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:44:47.834+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:44:47.834+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:44:47.836+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:44:47.836+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:44:47.836+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:44:47.877+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.047 seconds
[2023-06-01T18:45:17.985+0000] {processor.py:157} INFO - Started process (PID=68) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:45:17.986+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:45:17.987+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:45:17.987+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:45:17.989+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:45:17.988+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:45:17.989+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:45:18.029+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.046 seconds
[2023-06-01T18:45:48.134+0000] {processor.py:157} INFO - Started process (PID=77) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:45:48.136+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:45:48.137+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:45:48.137+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:45:48.138+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:45:48.138+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:45:48.139+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:45:48.178+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.046 seconds
[2023-06-01T18:46:18.287+0000] {processor.py:157} INFO - Started process (PID=86) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:46:18.289+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:46:18.290+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:46:18.289+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:46:18.291+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:46:18.291+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:46:18.292+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:46:18.331+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.046 seconds
[2023-06-01T18:46:48.435+0000] {processor.py:157} INFO - Started process (PID=95) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:46:48.438+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:46:48.438+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:46:48.438+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:46:48.440+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:46:48.439+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:46:48.440+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:46:48.479+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.046 seconds
[2023-06-01T18:47:18.593+0000] {processor.py:157} INFO - Started process (PID=104) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:47:18.595+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:47:18.595+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:47:18.595+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:47:18.597+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:47:18.597+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    import CsvDagWrapper
ModuleNotFoundError: No module named 'CsvDagWrapper'
[2023-06-01T18:47:18.597+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:47:18.636+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.045 seconds
[2023-06-01T18:47:23.662+0000] {processor.py:157} INFO - Started process (PID=106) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:47:23.664+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:47:23.664+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:47:23.664+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:47:23.666+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:47:23.665+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from .include.Csv2Delta.utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:47:23.666+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:47:23.677+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.017 seconds
[2023-06-01T18:47:53.491+0000] {processor.py:157} INFO - Started process (PID=23) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:47:53.492+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:47:53.493+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:47:53.493+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:47:53.495+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:47:53.494+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from .include.Csv2Delta.utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:47:53.495+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:47:53.541+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.053 seconds
[2023-06-01T18:48:23.602+0000] {processor.py:157} INFO - Started process (PID=32) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:48:23.604+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:48:23.605+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:48:23.605+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:48:23.606+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:48:23.606+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from .include.Csv2Delta.utils import CsvDagWrapper
ImportError: attempted relative import with no known parent package
[2023-06-01T18:48:23.606+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:48:23.647+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.047 seconds
[2023-06-01T18:48:52.704+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:48:52.705+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:48:52.706+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:48:52.705+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:48:53.063+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:48:53.063+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from include.Csv2Delta.utils import CsvDagWrapper
  File "/opt/airflow/dags/include/Csv2Delta/utils/CsvDagWrapper.py", line 15, in <module>
    import Config
ModuleNotFoundError: No module named 'Config'
[2023-06-01T18:48:53.064+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:48:53.074+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.371 seconds
[2023-06-01T18:49:31.215+0000] {processor.py:157} INFO - Started process (PID=23) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:49:31.216+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:49:31.216+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:49:31.216+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:49:31.515+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:49:31.515+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from include.Csv2Delta.utils import CsvDagWrapper
  File "/opt/airflow/dags/include/Csv2Delta/utils/CsvDagWrapper.py", line 15, in <module>
    import Config
ModuleNotFoundError: No module named 'Config'
[2023-06-01T18:49:31.515+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:49:31.525+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.313 seconds
[2023-06-01T18:50:02.371+0000] {processor.py:157} INFO - Started process (PID=45) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:50:02.372+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:50:02.372+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:50:02.372+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:50:02.724+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:50:02.723+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from include.Csv2Delta.utils import CsvDagWrapper
  File "/opt/airflow/dags/include/Csv2Delta/utils/CsvDagWrapper.py", line 15, in <module>
    import Config
ModuleNotFoundError: No module named 'Config'
[2023-06-01T18:50:02.724+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:50:02.735+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.365 seconds
[2023-06-01T18:50:33.484+0000] {processor.py:157} INFO - Started process (PID=66) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:50:33.485+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:50:33.485+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:50:33.485+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:50:33.802+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:50:33.801+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from include.Csv2Delta.utils import CsvDagWrapper
  File "/opt/airflow/dags/include/Csv2Delta/utils/CsvDagWrapper.py", line 15, in <module>
    import Config
ModuleNotFoundError: No module named 'Config'
[2023-06-01T18:50:33.802+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:50:33.813+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.331 seconds
[2023-06-01T18:51:04.605+0000] {processor.py:157} INFO - Started process (PID=87) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:51:04.606+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:51:04.607+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:51:04.607+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:51:04.939+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:51:04.939+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from include.Csv2Delta.utils import CsvDagWrapper
  File "/opt/airflow/dags/include/Csv2Delta/utils/CsvDagWrapper.py", line 15, in <module>
    import Config
ModuleNotFoundError: No module named 'Config'
[2023-06-01T18:51:04.940+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:51:04.949+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.347 seconds
[2023-06-01T18:51:33.911+0000] {processor.py:157} INFO - Started process (PID=23) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:51:33.913+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:51:33.913+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:51:33.913+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:51:33.917+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:51:33.916+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from include.Csv2Delta.utils import CsvDagWrapper
  File "/opt/airflow/dags/include/Csv2Delta/utils/CsvDagWrapper.py", line 15
    from include.Csv2Delta.utils.Config
                                       ^
SyntaxError: invalid syntax
[2023-06-01T18:51:33.918+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:51:33.955+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.047 seconds
[2023-06-01T18:52:04.089+0000] {processor.py:157} INFO - Started process (PID=32) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:52:04.093+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:52:04.093+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:52:04.093+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:52:04.408+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:52:04.407+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from include.Csv2Delta.utils import CsvDagWrapper
  File "/opt/airflow/dags/include/Csv2Delta/utils/CsvDagWrapper.py", line 16, in <module>
    import Csv2Delta
ModuleNotFoundError: No module named 'Csv2Delta'
[2023-06-01T18:52:04.408+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:52:04.419+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.332 seconds
[2023-06-01T18:52:35.217+0000] {processor.py:157} INFO - Started process (PID=54) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:52:35.218+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:52:35.218+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:52:35.218+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:52:35.542+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:52:35.541+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from include.Csv2Delta.utils import CsvDagWrapper
  File "/opt/airflow/dags/include/Csv2Delta/utils/CsvDagWrapper.py", line 16, in <module>
    import Csv2Delta
ModuleNotFoundError: No module named 'Csv2Delta'
[2023-06-01T18:52:35.542+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:52:35.552+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.337 seconds
[2023-06-01T18:53:05.366+0000] {processor.py:157} INFO - Started process (PID=23) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:53:05.366+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:53:05.367+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:53:05.367+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:53:05.723+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:53:05.722+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from include.Csv2Delta.utils import CsvDagWrapper
  File "/opt/airflow/dags/include/Csv2Delta/utils/CsvDagWrapper.py", line 17, in <module>
    from include.Csv2Delta.utils import SparkApp
  File "/opt/airflow/dags/include/Csv2Delta/utils/SparkApp.py", line 1, in <module>
    import Config
ModuleNotFoundError: No module named 'Config'
[2023-06-01T18:53:05.723+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:53:05.733+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.370 seconds
[2023-06-01T18:53:36.495+0000] {processor.py:157} INFO - Started process (PID=44) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:53:36.496+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:53:36.497+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:53:36.496+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:53:36.818+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:53:36.817+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from include.Csv2Delta.utils import CsvDagWrapper
  File "/opt/airflow/dags/include/Csv2Delta/utils/CsvDagWrapper.py", line 17, in <module>
    from include.Csv2Delta.utils import SparkApp
  File "/opt/airflow/dags/include/Csv2Delta/utils/SparkApp.py", line 1, in <module>
    import Config
ModuleNotFoundError: No module named 'Config'
[2023-06-01T18:53:36.819+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:53:36.830+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.338 seconds
[2023-06-01T18:54:07.608+0000] {processor.py:157} INFO - Started process (PID=65) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:54:07.610+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:54:07.610+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:54:07.610+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:54:07.913+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:54:07.913+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/WrapperDag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/WrapperDag.py", line 11, in <module>
    from include.Csv2Delta.utils import CsvDagWrapper
  File "/opt/airflow/dags/include/Csv2Delta/utils/CsvDagWrapper.py", line 17, in <module>
    from include.Csv2Delta.utils import SparkApp
  File "/opt/airflow/dags/include/Csv2Delta/utils/SparkApp.py", line 1, in <module>
    import Config
ModuleNotFoundError: No module named 'Config'
[2023-06-01T18:54:07.913+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:54:07.925+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.319 seconds
[2023-06-01T18:54:38.732+0000] {processor.py:157} INFO - Started process (PID=86) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:54:38.733+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:54:38.734+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:54:38.734+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:54:39.058+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:54:39.067+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.337 seconds
[2023-06-01T18:55:09.847+0000] {processor.py:157} INFO - Started process (PID=107) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:55:09.849+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:55:09.850+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:55:09.849+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:55:10.140+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:55:10.146+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.301 seconds
[2023-06-01T18:55:40.975+0000] {processor.py:157} INFO - Started process (PID=128) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:55:40.976+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:55:40.977+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:55:40.977+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:55:41.280+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:55:41.286+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.313 seconds
[2023-06-01T18:56:12.104+0000] {processor.py:157} INFO - Started process (PID=149) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:56:12.105+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:56:12.106+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:56:12.106+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:56:12.411+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:56:12.418+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.316 seconds
[2023-06-01T18:56:43.226+0000] {processor.py:157} INFO - Started process (PID=170) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:56:43.227+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:56:43.228+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:56:43.228+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:56:43.566+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:56:43.573+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.349 seconds
[2023-06-01T18:57:14.343+0000] {processor.py:157} INFO - Started process (PID=191) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:57:14.345+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:57:14.345+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:57:14.345+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:57:14.669+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:57:14.676+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.334 seconds
[2023-06-01T18:57:45.460+0000] {processor.py:157} INFO - Started process (PID=212) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:57:45.461+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:57:45.462+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:57:45.461+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:57:45.779+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:57:45.785+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.327 seconds
[2023-06-01T18:58:16.590+0000] {processor.py:157} INFO - Started process (PID=233) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:58:16.591+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:58:16.592+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:58:16.591+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:58:16.901+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:58:16.907+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.319 seconds
[2023-06-01T18:58:47.714+0000] {processor.py:157} INFO - Started process (PID=254) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:58:47.715+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:58:47.716+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:58:47.716+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:58:48.025+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:58:48.031+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.319 seconds
[2023-06-01T18:59:18.832+0000] {processor.py:157} INFO - Started process (PID=275) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:59:18.832+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:59:18.833+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:59:18.833+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:59:19.141+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:59:19.150+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.320 seconds
[2023-06-01T18:59:49.937+0000] {processor.py:157} INFO - Started process (PID=296) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:59:49.938+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T18:59:49.939+0000] {logging_mixin.py:149} INFO - [2023-06-01T18:59:49.939+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:59:50.246+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T18:59:50.253+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.318 seconds
[2023-06-01T19:00:21.054+0000] {processor.py:157} INFO - Started process (PID=317) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:00:21.055+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T19:00:21.055+0000] {logging_mixin.py:149} INFO - [2023-06-01T19:00:21.055+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:00:21.351+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:00:21.359+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.307 seconds
[2023-06-01T19:00:52.176+0000] {processor.py:157} INFO - Started process (PID=338) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:00:52.177+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T19:00:52.177+0000] {logging_mixin.py:149} INFO - [2023-06-01T19:00:52.177+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:00:52.495+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:00:52.502+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.328 seconds
[2023-06-01T19:01:23.284+0000] {processor.py:157} INFO - Started process (PID=359) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:01:23.285+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T19:01:23.286+0000] {logging_mixin.py:149} INFO - [2023-06-01T19:01:23.286+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:01:23.585+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:01:23.591+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.310 seconds
[2023-06-01T19:01:54.396+0000] {processor.py:157} INFO - Started process (PID=380) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:01:54.398+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T19:01:54.398+0000] {logging_mixin.py:149} INFO - [2023-06-01T19:01:54.398+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:01:54.691+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:01:54.698+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.303 seconds
[2023-06-01T19:02:25.509+0000] {processor.py:157} INFO - Started process (PID=401) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:02:25.510+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T19:02:25.511+0000] {logging_mixin.py:149} INFO - [2023-06-01T19:02:25.511+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:02:25.812+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:02:25.819+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.312 seconds
[2023-06-01T19:02:56.621+0000] {processor.py:157} INFO - Started process (PID=422) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:02:56.622+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T19:02:56.623+0000] {logging_mixin.py:149} INFO - [2023-06-01T19:02:56.623+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:02:56.957+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:02:56.964+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.345 seconds
[2023-06-01T19:03:27.733+0000] {processor.py:157} INFO - Started process (PID=443) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:03:27.735+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T19:03:27.735+0000] {logging_mixin.py:149} INFO - [2023-06-01T19:03:27.735+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:03:28.054+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:03:28.060+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.328 seconds
[2023-06-01T19:03:58.840+0000] {processor.py:157} INFO - Started process (PID=464) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:03:58.841+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T19:03:58.842+0000] {logging_mixin.py:149} INFO - [2023-06-01T19:03:58.842+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:03:59.145+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:03:59.151+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.314 seconds
[2023-06-01T19:04:29.944+0000] {processor.py:157} INFO - Started process (PID=485) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:04:29.945+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T19:04:29.946+0000] {logging_mixin.py:149} INFO - [2023-06-01T19:04:29.946+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:04:30.260+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:04:30.266+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.323 seconds
[2023-06-01T19:05:01.051+0000] {processor.py:157} INFO - Started process (PID=506) to work on /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:05:01.053+0000] {processor.py:826} INFO - Processing file /opt/airflow/dags/WrapperDag.py for tasks to queue
[2023-06-01T19:05:01.053+0000] {logging_mixin.py:149} INFO - [2023-06-01T19:05:01.053+0000] {dagbag.py:541} INFO - Filling up the DagBag from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:05:01.362+0000] {processor.py:838} WARNING - No viable dags retrieved from /opt/airflow/dags/WrapperDag.py
[2023-06-01T19:05:01.370+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/WrapperDag.py took 0.321 seconds
